stacks/openshift/application/pipeline/kustomization.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+- ../../../../pipeline/minio/installs/generic
+- ../../../../pipeline/mysql/installs/generic
+- ../../../../pipeline/installs/multi-user
```

Comment:
Why are we using multi-user kfp over here? In this PR's kfdef, there's no authentication services and istio sidecar injection to provide istio auth. Thus, users cannot run pipelines on the multi-user kfp because they cannot authenticate themselves.

Reply:
Here is how to make it run with single user, https://github.com/nakfour/manifests/compare/openshift_stack...Tomcli:openshift-refactor

Reply:
We disabled istio-injection, as posted in a comment before. We at the moment cannot pull pipeline into applications because of its dependency on profiles. We discussed this before

Reply:
Yes, I understand you want to disable istio-injection, so we should using the single user kfp isn't it? With the single user kfp, there shouldn't be a dependency on profiles.

Reply:
`../../../../pipeline/installs/generic` is the single user mode kfp

Reply:
Ok but. we would like users to run pipelines in their own namespace generated by the profile controller. Isn't that the whole point of multi-user pipelines

Reply:
Sure, but in that case you will need to provide authentication services and istio sidecar injection to authenticate incoming user traffics. And the current istio sidecar injection is not ideal for openshift.

Reply:
right now with this deployment, you cannot submit pipelines as a user.

Reply:
Yes switched pipeline to generic and pulled out into kfdef